products/caregivers/Improve Facility Selection/10-10 Health Apps-10-10CG-[date].md

```diff
@@ -2,71 +2,109 @@
 
 ## Instructions
 
-Your document should be brief and high-level.  Please keep it to a single page.  Focus on the high level and link to supporting material where appropriate; this is _not_ a detailed engineering spec.
+Your document should be brief and high-level. Please keep it to a single page. Focus on the high level and link to supporting material where appropriate; this is _not_ a detailed engineering spec.
 
-Some of the items below may not apply to your work--that's okay.  You may not be able to fill in some items that _do_ apply to your work--that's also okay.  If you don't have answers, please come ready to ask questions.
+Some of the items below may not apply to your work--that's okay. You may not be able to fill in some items that _do_ apply to your work--that's also okay. If you don't have answers, please come ready to ask questions.
 
 ## Checklist
 
 - Product description
-    + Brief overview of motivation for the change from an engineering & security point of view
-    + [Link to Collaboration Cycle Request issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/51980)
+  - Brief overview of motivation for the change from an engineering & security point of view
+       - Initially, the facility selection page was based on U.S. State dropdown selection, then a facility dropdown selection, derived from a static JSON file. This caused our facilities list to easily be out-of-date and inaccurate.  We want to provide applicants with the option to search by city, state or postal code of any given facility, using the latest data from the Facilities API.
+  - [Link to Collaboration Cycle Request issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/51980)
 - UX design description
-    + For user-facing changes, link to UX prototype or wireframes if available
-         + [Figma designs](https://www.figma.com/design/TxXD5bGUOhbHHWLb85GPjK/10-10CG?node-id=2478-18708&node-type=SECTION&t=LIbnyXIFEnPa5jZz-0)
-    + Call out any engineering challenges; UX is reviewed in the [Design Intent meeting][DI]
```

Comment:
Did we have any engineering challenges?

Reply:
Not really? Overall, it is a pretty simple architecture in that it introduces some apis and new fields. Anything too specific seems too technical. Most of the complexity feels like it is in the designs tbh, since we are still trying to figure out how to best make that aspect work. _Maybe_ the review screen since we had to do some custom stuff there.

Reply:
I also updated this.